src/main/app/claims/models/settlement.ts

```diff
@@ -79,6 +79,14 @@ export class Settlement {
     return this.partyStatements && this.partyStatements.some(statement => statement.type === 'COUNTERSIGNATURE')
   }
 
+  isOfferRejectedByDefendant (): boolean {
+    const statement: PartyStatement = this.partyStatements
+      .filter(o => o.type === StatementType.REJECTION.value && o.madeBy === MadeBy.DEFENDANT.value)
+      .pop()
```

Comment:
Readability might be improved with two `.filter()`s instead of a single conjunction one.